app/helpers/application_helper.rb

```diff
@@ -6,9 +6,9 @@ def current_user?(user)
   end
 
   def lid?
-    return unless logged_in?
+    return if logged_in?
 
-    redirect_to root_path, notice: "Je account mag niet bij deze resource." unless current_user&.lid? || current_user&.olid?
+    redirect_to root_path, notice: "Je account mag niet bij deze resource." unless current_user&.lid?
```

Comment:
Dit gaat niet goed,` current_user&.lid? ` wordt nu nooit gecheckt omdat `current_user&.active?` true geeft op `logged in?` 

Met deze changes is dit dus effectief alleen een logged_in? check en niet een lid? check.